oidc-controller/api/authSessions/models.py

```diff
@@ -27,8 +27,13 @@ class AuthSessionBase(BaseModel):
     pyop_auth_code: str
     response_url: str
     presentation_request_msg: dict | None = None
+    connection_id: str | None = None  # NEW: Track connection ID
```

Comment:
Should index on this I think? In crud get_by_connection_id it's looking up based on connection_id